stockfish/models.py

```diff
@@ -706,11 +706,18 @@ def is_move_correct(self, move_value: str) -> bool:
         self.info = old_self_info
         return is_move_correct
 
-    def get_wdl_stats(self) -> Optional[List[int]]:
+    def get_wdl_stats(
+        self, get_as_tuple: bool = False
+    ) -> Union[list[int] | tuple[int, ...] | None]:
```

Comment:
For this we could do `tuple[int, int, int]`, since it's guaranteed the tuple will have 3 elements.